.fixtures.yml

```diff
@@ -1,8 +1,6 @@
 fixtures:
   repositories:
     concat: 'git://github.com/puppetlabs/puppetlabs-concat.git'
-    stdlib:
-      repo: 'git://github.com/puppetlabs/puppetlabs-stdlib.git'
-      ref: '3.0.0'
+    stdlib: 'git://github.com/puppetlabs/puppetlabs-stdlib.git'
```

Comment:
yes, but it has a Modulefile, and it says:
```
dependency 'puppetlabs/concat', '> 1.0.0'
dependency 'puppetlabs/stdlib', '>= 3.0.0'
```
anyway. I suggest, as always, we release 1.0.0 as initial version, as soon as travis is running — and happy :D